nixos/modules/config/users-groups.nix

```diff
@@ -681,6 +681,7 @@ in {
 
     environment.profiles = [
       "$HOME/.nix-profile"
+      "\${XDG_STATE_HOME:-$HOME/.local/state}/nix/profile"
```

Comment:
Weird that it has to be escaped with a \ but it's necessary `''` did not work.

Line is generated ok

```
export NIX_PROFILES="/run/current-system/sw /nix/var/nix/profiles/default /etc/profiles/per-user/$USER ${XDG_STATE_HOME:-$HOME/.local/state}/nix/profile $HOME/.nix-profile"
```

Reply:
Escaping with `''` is for multi-line strings, single line strings just use `\` AFAIK.

Reply:
MatthewCash is correct, see https://nixos.org/manual/nix/stable/language/values#type-string

Reply:
Yeah I had to try both variations